tests/chainer_tests/functions_tests/math_tests/test_basic_math.py

```diff
@@ -216,6 +216,58 @@ def test_rpow_double_backward_gpu(self):
         self.double_backward_gpu(lambda x, y: y.__rpow__(x))
 
 
+@testing.parameterize(*testing.product({
+    'shape': [(3, 2), ()],
+    'dtype': [numpy.float16, numpy.float32, numpy.float64],
+}))
+class TestBinaryOpTuple(unittest.TestCase):
```

Comment:
I think `TestBinaryOpTuple` is not appropriate because what is tested is multiple-operand arithmetics.
How about `TestMultipleAdd` or something like that?

Reply:
Yes. I agree.